lib/cli.js

```diff
@@ -66,7 +67,14 @@ module.exports = function broccoliCLI(args) {
         watcher.on('buildSuccess', () => outputTree.sync());
       }
 
-      actionPromise = broccoli.server.serve(watcher, options.host, parseInt(options.port, 10));
+      actionPromise = broccoli.server.serve(
+        watcher,
+        options.host,
```

Comment:
Its starting to feel like maybe we should switch to a config argument instead of just the raw args:

```
broccoli.server.serve({
  watcher,
  host: options.host,
  port: parseInt(options.port, 10),
  ui
});
```

We don't have to do this as apart of this PR but it feels weird to have to pass multiple `undefined`'s

Reply:
yes, this would be a breaking change so we likely need to wait for v4.